Homework4/task2.py

```diff
@@ -0,0 +1,17 @@
+# 2. Города
+# Дан список стран и городов каждой страны. Затем даны названия городов.
+# Для каждого города укажите, в какой стране он находится.
+
+dct = {}
+for the_list in range(int(input("Enter the number of lists: "))):
+    country, *cities = input("Enter a list: ").split()
+    dct[country] = cities
+for city in range(int(input("Enter the number of cities: "))):
+    the_city = input("Enter the city: ")
+    count = 0
+    for country, cities in dct.items():
```

Comment:
O(n*m) solution (n - num of requested cities m - size of dict)
in fact we miss dict advantages here